R/databricks_connection.R

```diff
@@ -1,10 +1,11 @@
 databricks_connection <- function(config, extensions) {
   tryCatch({
     callSparkR <- get("callJStatic", envir = asNamespace("SparkR"))
+    # In Databricks notebooks, this variable is in the default namespace
```

Comment:
The comment can be more general. On Databricks (notebooks or RStudio) `DATABRICKS_GUID` is available in the global namespace.